### ISSUE TITLE
In 200% zoom, 'Got feedback' tab item button is getting overlapped by some other text after receiving keyboard focus.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1375]
**Test Environment:**
OS Version: Windows 11 Build 22504.1010
Browser: Edge Dev Version 98.0.1093.6 (Official build) dev (64-bit)
URL: Graph Explorer - Microsoft Graph
Screen Reader: Narrator

**Pre-Requisite:**
1.Setting Display Resolution to '1280*768
2.Select Zoom settings to '200%'  for Anaheim dev browser.
3.Refresh the page.

**Repro Steps:**

1. Open above URL in Edge browser.
2. Login with valid credential.
3. Navigate to the 'Access token' button using tab key.
4. Observe if the 'Access token' text is getting overlapped or not.

**Actual Result:**
In 200% zoom, 'Access token' tab item button is getting overlapped by some other text after receiving keyboard focus.

**Expected Result:**
In 200% zoom, 'Access token' tab item button should be properly visible after receiving keyboard focus.

**User Impact:** 
Low vision users will be impacted as they will not be able to see the control if it will get overlapped by other.

**MAS Reference:**
[Resize Text - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.04.04)

**Screenshots**

![image](https://user-images.githubusercontent.com/9458079/152305903-3305e95d-add8-490d-86fe-e33c8fbeb492.png)

**Comment by ElinorW** [comment]
tracking this here https://github.com/microsoft/fluentui/issues/20664

**Comment by ElinorW** [comment]
No longer an issue as the Feedback button was moved in this PR 
https://github.com/microsoftgraph/microsoft-graph-explorer-v4/pull/1455
